src/cdk/overlay/overlay.spec.ts

```diff
@@ -1065,6 +1066,65 @@ describe('Overlay', () => {
   });
 });
 
+// Caretaker note: this is moved to a separate `describe` block because the first `describe` block
```

Comment:
Saying "caretaker note" here isn't necessary, this is just a regular code comment. We use "Caretaker note" in PR comments to mention something that might come up during the process of merging the PR and syncing it into Google's internal code repo.